models/ed/R/met2model.ED2.R

```diff
@@ -267,7 +267,7 @@ for(year in start_year:end_year) {
 
   ## write DRIVER file
   sites <- 1
-  metgrid <- c(1,1,1,1,lon,lat)
+  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
```

Comment:
I believe the lat/lon seemed to work for me best. The floor function would round it off and I would end up with case where the lat/lon would fall outside of the point area that ED uses. For example 0.9 would become 0, and ed would use -0.5 to 0.5 (or something along those lines).